### ISSUE TITLE
Boostrap: should we not compare with the mean of the resampled medians?

### ISSUE THREAD

**Issue by reitzig** [swirldev/swirl_courses#333]
In Statistical_Inference/Resampling, we are asked several times to compare the median of the resampled medians with the median of the original data set.

But is not the *mean* of the resampled statistics what we want to compare to the original statistic?

**Comment by seankross** [comment]
If I recall correctly the mean and the median converge in a distribution of sample medians.

**Comment by reitzig** [comment]
My issue is that the course seems to suggest that `stat(resampledStats) == stat(data) +- epsilon` is what you should expect. For any statistic but median or mean, this will go horribly wrong.

**Comment by seankross** [comment]
Would you mind clarifying what you mean? I feel like discussing the
situations where bootstrapping is inappropriate is outside the scope of
this course.

**Comment by reitzig** [comment]
The course very clearly states that we can use bootstrapping methods to gain information about the distribution of analytically intractable statistics. Say we want to investigate statistic `f` of some data `data`. We apply the bootstrap and obtain several thousand values `f(resampledData)`.

The unwary student may extrapolate from the course, compute `f(f(resampledData)) - f(data)` and look for the result to be close to zero. This is clearly fallacious; they should expect `mean(f(resampledData)) - f(data)` to be close to zero.

(The example in the course is with `f = median`.)

**Comment by seankross** [comment]
I don't believe that the bootstrap can be generalized to finding the average of any statistic that the method is applied to. It's proven to be useful for constructing confidence intervals and we discuss that in this lesson.

**Comment by reitzig** [comment]
Then the course is misleading in more than one regard. :/
(I'm taking the Data Science Specialization on Coursera, so I watched the lectures corresponding to this swirl lesson before.)
